src/daidepp/grammar/grammar.py

```diff
@@ -51,7 +51,7 @@
     "try": '"TRY" lpar try_tokens (ws try_tokens)* rpar',
     "huh": '"HUH" lpar press_message rpar',
     "prp": '"PRP" lpar arrangement rpar',
-    "aly_vss": '"ALY" lpar power (ws power)+ rpar "VSS" lpar power (ws power)+ rpar',
+    "aly_vss": '"ALY" lpar power (ws power)* rpar "VSS" lpar power (ws power)* rpar',
```

Comment:
```suggestion
    "aly_vss": '"ALY" lpar power (ws power)+ rpar "VSS" lpar power (ws power)* rpar',
```
A single power cannot enter an alliance without another power.